token/sdk/dig/sdk.go

```diff
@@ -128,28 +127,16 @@ func (p *SDK) Install() error {
 				new(common2.TokenManagementServiceProvider),
 			),
 		),
-		p.Container().Provide(func(dh *db2.DriverHolder) *ttxdb.Manager {
-			return ttxdb.NewManager(dh, "ttxdb.persistence", "db.persistence")
-		}),
+		p.Container().Provide(ttxdb.NewManager),
 		p.Container().Provide(digutils.Identity[*ttxdb.Manager](), dig.As(new(ttx.DBProvider), new(network2.TTXDBProvider))),
-		p.Container().Provide(func(dh *db2.DriverHolder) *tokendb.Manager {
-			return tokendb.NewManager(dh, "tokendb.persistence", "db.persistence")
-		}),
-		p.Container().Provide(func(dh *db2.DriverHolder) *tokendb.NotifierManager {
-			return tokendb.NewNotifierManager(dh, "tokendb.persistence", "db.persistence")
-		}),
+		p.Container().Provide(tokendb.NewManager),
+		p.Container().Provide(tokendb.NewNotifierManager),
 		p.Container().Provide(digutils.Identity[*tokendb.Manager](), dig.As(new(tokens.DBProvider))),
 		p.Container().Provide(NewDriverHolder),
-		p.Container().Provide(func(dh *db2.DriverHolder) *auditdb.Manager {
```

Comment:
The config prefix in now moved in the manager instead of the SDK to be aligned with the SDK